mappings/net/minecraft/structure/StructurePieceWithDimensions.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS cdr net/minecraft/structure/StructurePieceWithDimensions
+	FIELD a width I
+	FIELD b height I
+	FIELD c depth I
+	FIELD d hPos I
```

Comment:
hPos?

Reply:
It's the NBT name

Reply:
I think it's fine if it's the NBT name, myself